### PR TITLE
feature(i18n): added function to check if a language key exists

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: engine/classes/Elgg/I18n/Translator.php [hunk 1]
```diff
@@ -391,6 +391,35 @@ function getMissingLanguageKeys($language) {
...
+		if (($language !== 'en') && !array_key_exists($language, $this->CONFIG->translations)) {
+			// Ensure that all possible translations are loaded
+			$this->reloadAllTranslations();
+		}
+	
+		if (!array_key_exists($language, $this->CONFIG->translations)) {
+			return false;
+		}
+	
+		if (array_key_exists($key, $this->CONFIG->translations[$language])) {
```
Comment (ewinslow) [inline engine/classes/Elgg/I18n/Translator.php]: No need for if. Just return directly.

**Comment by ewinslow** [comment]
I'm fine with this.

@mrclay had opinion iirc

File: engine/classes/Elgg/I18n/Translator.php (hunk 1 above)
Comment (mrclay) [inline engine/classes/Elgg/I18n/Translator.php]: Fine as is. Nicely consistent.

**Comment by mrclay** [comment]
Seems like the simplest solution.

**Comment by jdalsem** [comment]
now directly returns array_key_exists

**Comment by ewinslow** [comment]
Fantastic. LGTM once build failure are fixed

**Pull request merged**